groups/stack/module-ecs-services/officer-filing-api.tf

```diff
@@ -0,0 +1,105 @@
+locals {
+  service_name = "officer-filing-api"
+  officer_filing_api_proxy_port = 11000 # local port number defined for proxy target of officer-filing-api service sitting behind eric
+}
+
+resource "aws_ecs_service" "officer-filing-api-ecs-service" {
+  name            = "${var.environment}-${local.service_name}"
+  cluster         = var.ecs_cluster_id
+  task_definition = aws_ecs_task_definition.officer-filing-api-task-definition.arn
+  desired_count   = 1
+  depends_on      = [var.officer-filing-api-lb-arn]
+  load_balancer {
+    target_group_arn = aws_lb_target_group.officer-filing-api-target_group.arn
+    container_port   = var.officer_filing_api_application_port
+    container_name   = "eric" # [ALB -> target group -> eric -> officer filing api] so eric container named here
+  }
+}
+
+locals {
```

Comment:
Please consider splitting `locals` in to their own file, e.g. `locals.tf` to aid readability.

Reply:
done a locals file here too.